README.md

```diff
@@ -35,3 +35,10 @@ host network and need network interfaces to be configured beforehand.
 
 To add a custom configuration to the container, a volume may be mounted
 to `/etc/capwap-dp.conf`.
+
+Running
+-------
+
+For running under unprivileged user should add capabilities:
```

Comment:
that doesn't sound like proper English. Also, when giving hints like this, please explain first what is need and then the command as example. Throwing random commands at the reader doesn't help them understand why this is done.